docs/customization.md

```diff
@@ -1,5 +1,41 @@
 # Customize your instance
 
+## Allowed email domains (new in 1.5.8)
+
+You may restrict the email domains that can be used for creating new accounts by specifying a file containing one line
+per domain. Matches are performed in a case-insensitive manner.
```

Comment:
It's preferable to use the terminology "email providers" instead of "email domains" since this is less technical.